DESCRIPTION

```diff
@@ -3,7 +3,7 @@ Type: Package
 Title: GENetic EStimation and Inference in Structured samples
         (GENESIS): Statistical methods for analyzing genetic data from
         samples with population structure and/or relatedness
-Version: 2.19.8
+Version: 2.19.9
```

Comment:
Should be version 2.21.1 (I just pulled the latest version number after release, sorry I forgot to do this earlier). Also update date.

Reply:
Thanks, fixed!